sssom/sparql_util.py

```diff
@@ -84,10 +84,22 @@ def query_mappings(config: EndpointConfig) -> MappingSetDataFrame:
 
 
 def curiefy_row(row: Mapping[str, str], config: EndpointConfig) -> Dict[str, str]:
+    """Assign a contracted URI ("dc") if Mapping object dictionary value is "http://purl.org/dc/terms/".
+
+    :param row: Mapping object row
+    :param config: Configuration
+    :return: Dictionary of CURIEs
+    """
     return {k: contract_uri(v, config) for k, v in row.items()}
 
 
 def contract_uri(uristr: str, config: EndpointConfig) -> str:
+    """Contract URI. For e.g. "http://purl.org/dc/terms/" => "dc".
```

Comment:
Here's an example of a more informative docstring:

```suggestion
    """Replace the URI with a prefix based on the prefix map in the given configuration.
```

It says what the function does and how it does it. Perhaps consider renaming the variables, too